lib/litmus_paper/agent_check_handler.rb

```diff
@@ -1,8 +1,19 @@
 module LitmusPaper
   class AgentCheckHandler
     def self.handle(service)
+      @cache ||= LitmusPaper::Cache.new(
+        LitmusPaper.cache_location,
+        "litmus_cache",
```

Comment:
I'm not sure if the agent_check_server runs on the same machine or simultaneously as the regular litmus_paper, but it looks like we'd be storing different types of values for the same keys. We may want to change the namespace here to avoid collisions.

Reply:
@pblesi good catch, it definitely can run on the same host, and does in our usage.

Reply:
My thought had been that we might wish for the caches to be shared between the web processes and the agent-check processes (something something efficiency) ... but it's probably not worth it. I'll change.